bridge/contracts/nftbridge/INFTBridge.sol

```diff
@@ -12,6 +12,11 @@ interface INFTBridge {
         uint32 logIndex;
     }
 
+    // struct NFTcrossAddresses {
```

Comment:
I think we can delete this as we wont use it

Reply:
Done at https://github.com/rsksmart/tokenbridge/pull/242/commits/9f860e33629ac300601c52918e9af3ee15e85572